pdc/apps/utils/__init__.py

```diff
@@ -5,7 +5,8 @@
 #
 from django.conf import settings
 
-from .messaging import DummyMessenger, KombuMessenger, FedmsgMessenger, ProtonMessenger
+from .messaging import (DummyMessenger, KombuMessenger, FedmsgMessenger,
+                        ProtonMessenger, StompMessenger)
```

Comment:
ProtonMessenger just like KombuMessenger, it works, just not support multi endpoints failover, so I suggest we leave it there but remove all related items from spec and requirements.